<?xml version="1.0"?>
<launch>
    <!-- joy node -->
    <node respawn="true" pkg="joy" type="joy_node" name="ros_joy_node">
        <param name="dev" type="string" value="/dev/input/js0"/>
        <param name="deadzone" value="0.12"/> 
    </node>

    <!-- Axes -->
    <param name="axis_linear" value="1" type="int"/>
    <param name="axis_angular" value="3" type="int"/>
    <param name="button_deadman_switch" value="2" type="int"/>

    <param name="scale_linear" value="1.0" type="double"/>
    <param name="scale_angular" value="1.0" type="double"/>

    <!-- JOY TO COMMAND VEL -->
    <node  pkg="kamtoa_teleop" type="kamtoa_joystick" name="kamtoa_joystick" output="screen" />
</launch>
